data/autoyast_containers.xml.ep

```diff
@@ -65,96 +73,68 @@
-          <format config:type="boolean">true</format>
-          <filesystem config:type="symbol">ext2</filesystem>
-          % }
-          % if ($check_var->('UEFI', '1')) {
           <filesystem config:type="symbol">vfat</filesystem>
-          % }
-          % if ($check_var->('ARCH', 's390x')) {
-          <mount>/boot/zipl</mount>
-          <fstopt>acl,user_xattr</fstopt>
-          % }
-          % if ($check_var->('UEFI', '1')) {
-          <mount>/boot/efi</mount>
+          <format config:type="boolean">true</format>
           <fstopt>umask=0002,utf8=true</fstopt>
-          % }
-          % unless ($check_var->('ARCH', 's390x') or $check_var->('ARCH', 'ppc64le') or $check_var->('UEFI', '1')) {
-          <mount>/boot</mount>
-          % }
-          <mountby config:type="symbol">path</mountby>
-          % if ($check_var->('ARCH', 'ppc64le')) {
-          <partition_id config:type="integer">65</partition_id>
-          % }
-          % unless ($check_var->('ARCH', 'ppc64le') or $check_var->('UEFI', '1')) {
-          <partition_id config:type="integer">131</partition_id>
-          % }
-          % if ($check_var->('UEFI', '1')) {
-          <partition_id config:type="integer">259</partition_id>
-          % }
-          <partition_nr config:type="integer">1</partition_nr>
-          <partition_type>primary</partition_type>
-          <size>500M</size>
+          <mount>/boot/efi</mount>
         </partition>
+        % }
         <partition>
+          <mountby config:type="symbol">device</mountby>
           <create config:type="boolean">true</create>
           <filesystem config:type="symbol">swap</filesystem>
           <format config:type="boolean">true</format>
+          <fstopt>defaults</fstopt>
           <mount>swap</mount>
-          <mountby config:type="symbol">uuid</mountby>
-          <partition_id config:type="integer">130</partition_id>
-          <partition_nr config:type="integer">2</partition_nr>
-          <partition_type>primary</partition_type>
-          <size>2G</size>
         </partition>
         <partition>
+          <mountby config:type="symbol">device</mountby>
           <create config:type="boolean">true</create>
+          <filesystem config:type="symbol">btrfs</filesystem>
           <format config:type="boolean">true</format>
+          <fstopt>defaults</fstopt>
           <mount>/</mount>
-          <mountby config:type="symbol">uuid</mountby>
-          <partition_id config:type="integer">131</partition_id>
-          <partition_nr config:type="integer">3</partition_nr>
-          <partition_type>primary</partition_type>
-          <size>max</size>
         </partition>
       </partitions>
-      <pesize/>
-      <type config:type="symbol">CT_DISK</type>
       <use>all</use>
     </drive>
   </partitioning>
+  % }
+  <firewall>
+    <enable_firewall config:type="boolean">false</enable_firewall>
+    <start_firewall config:type="boolean">false</start_firewall>
+  </firewall>
+  <report>
```

Comment:
do we really need this part ? usually I am dropping this ...

Reply:
I added it to avoid this situation https://bugzilla.suse.com/show_bug.cgi?id=1187176